openslides_backend/action/actions/meeting_user/create.py

```diff
@@ -38,3 +40,33 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 f"MeetingUser instance with user {instance['user_id']} and meeting {instance['meeting_id']} already exists"
             )
         return super().update_instance(instance)
+
+    def get_history_information(self) -> Optional[HistoryInformation]:
+        information = {}
+        for instance in self.instances:
+            instance_information = []
+            if "group_ids" in instance:
+                if len(instance["group_ids"]) == 1:
+                    instance_information.extend(
+                        [
+                            "Participant added to group {} in meeting {}",
+                            fqid_from_collection_and_id(
+                                "group", instance["group_ids"][0]
+                            ),
+                        ]
+                    )
+                else:
+                    instance_information.append(
+                        "Participant added to multiple groups in meeting {}",
+                    )
+            else:
+                instance_information.append(
+                    "Participant added to meeting {}",
+                )
+            instance_information.append(
```

Comment:
Why don't you use a format, where the placeholders are replaced by their respective values?

Reply:
I do, but the replacement is done in the client: every `{}` in the "main string" refers to a following replacement, which is interpreted by the client as an Fqid of which the title is then inserted.